chromium_src/BUILD.gn

```diff
@@ -400,10 +404,18 @@ source_set("browser") {
         "//chrome/browser/extensions/api/spellcheck/spellcheck_api.h",
         "//chrome/browser/extensions/api/tabs/tabs_constants.cc",
         "//chrome/browser/extensions/api/tabs/tabs_constants.h",
+        "//chrome/browser/extensions/api/tabs/windows_util.cc",
+        "//chrome/browser/extensions/api/tabs/windows_util.h",
         "//chrome/browser/extensions/chrome_extension_api_frame_id_map_helper.cc",
         "//chrome/browser/extensions/chrome_extension_api_frame_id_map_helper.h",
+        "//chrome/browser/extensions/chrome_extension_function_details.cc",
+        "//chrome/browser/extensions/chrome_extension_function_details.h",
         "//chrome/browser/extensions/event_router_forwarder.cc",
         "//chrome/browser/extensions/event_router_forwarder.h",
+        "//chrome/browser/extensions/window_controller.cc",
```

Comment:
this looks like a potential problem to me because along with windows_util above. None of these methods will work correctly without code so anything that uses them will get the wrong result. I think we need to look at the usage and possibly override with our own implementation